web/src/scripts/actions/tabActions.js

```diff
@@ -75,6 +75,51 @@ export const closeAllTabs = (containerId) => async (dispatch) => {
   dispatch({type: at.CLOSE_ALL_TABS, payload: {containerId}})
 }
 
+export const focusAdjacentTab = (containerId, iterator, options = {}) => async (dispatch, getState) => {
+  const tabs = getState().ui.tabs
+  const { groups, focusedGroupIndex } = _.get(tabs, `${containerId}`)
+  const focusedGroup = groups[focusedGroupIndex]
+  const focusedTabIndex = focusedGroup.tabIds.indexOf(focusedGroup.focusedTabId)
+  const newIndex = iterator(focusedTabIndex)
```

Comment:
It's a bit confusing to have the API take a function here. Maybe better to split into separate use cases